addon/components/goal/core-team.js

```diff
@@ -3,35 +3,40 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   layout,
-  _getPO: Ember.computed('team', function() {
-    let po = this.get('team').po;
-    if(po){
-      return po;
-    }
-      return "?";
-    }),
+  poFlag: false,
+  hdFlag: false,
+  hfeFlag: false,
+  hbeFlag: false,
+  _po: "?",
```

Comment:
Is it possible to automatically display a "?" instead of hardcoding it everywhere?